src/wallet/wallet2.h

```diff
@@ -2206,8 +2201,11 @@ namespace boost
           x.rct_config = { use_bulletproofs ? rct::RangeProofBulletproof : rct::RangeProofBorromean, 0 };
         return;
       }
-      a & x.v4_allow_tx_types;
```

Comment:
This whole kinda thing is almost useless, given its TX construction data which only gets serialized if you're trying to offline sign a TX. You'd have to open a view only wallet, create the transaction on our v5 wallet. Then download v6 and try to sign the constructed transaction to no avail.

People aren't holding onto constructed transactions for later signing either because then you'd fail sanity checks if that constructed transaction was sitting in storage for a long time.

So I didn't give much consideration to writing it bulletproof. Though that selected_transfers should not have been deleted. I can re-do the necessary fields.

Reply:
Ah, okay, makes sense.  I think it's fine to not support signing across mixed wallet versions.